src/addons/dragAndDrop/WeekWrapper.js

```diff
@@ -155,7 +155,7 @@ class WeekWrapper extends React.Component {
         return
       }
 
-      start = dates.min(dates.add(end, -1, 'day'), start)
+      start = dates.min(dates.add(end, 0, 'day'), start)
```

Comment:
i think this is the same as `end`? does adding nothing have an effect here?

Reply:
when we decrease a bar that time it will make possible to decrease till single day bar (same as start day).

Reply:
yes it is same as end. When we want to set bar for one day that time start and end will be same i think.
So review it and accept if it is ok.

Reply:
if its the same, lets remove the `dates.add()` and just use `end` same for the line above with `start`

Reply:
yes we can. If we can create new branch or something like for beta/alpha version so main version would not affect. May be we can get an issue after this change.

Reply:
has this issue been resolved? I am having the same bug. Cannot shring the event to one day unless the start date (left drag) has been modified. then the event is allowed to reduce to one day.